docs/template-__PROVIDER__-ensurabilitytest.rb

```diff
@@ -94,8 +94,7 @@
     on(master, __PROVIDERLIB__.create_<PROVIDER>_manifest_present)
 
     # Expected exit_code is 2 since this is a puppet agent cmd with change.
-    cmd_str = UtilityLib.get_namespace_cmd(agent, UtilityLib::PUPPET_BINPATH +
-      'agent -t', options)
+    cmd_str = UtilityLib::PUPPET_BINPATH + 'agent -t'
```

Comment:
Good to know. Honestly removing the get_namespace_cmd calls was just a bunch of search-and-replace - not a lot of work.